integrations/cassandra/src/metrics_definition.go

```diff
@@ -142,10 +151,6 @@ var columnFamilyDefinition = map[string][]interface{}{
 	"query.readLatency999thPercentileMilliseconds":  {"org.apache.cassandra.metrics:type=Table,name=ReadLatency,attr=999thPercentile", metric.GAUGE},
 	"db.allMemtablesOnHeapSizeBytes":                {"org.apache.cassandra.metrics:type=Table,name=AllMemtablesHeapSize,attr=Value", metric.GAUGE},
 	"db.allMemtablesOffHeapSizeBytes":               {"org.apache.cassandra.metrics:type=Table,name=AllMemtablesOffHeapSize,attr=Value", metric.GAUGE},
-
-	"db.keyspace":                {"keyspace", metric.ATTRIBUTE},
-	"db.columnFamily":            {"columnFamily", metric.ATTRIBUTE},
-	"db.keyspaceAndColumnFamily": {"keyspaceAndColumnFamily", metric.ATTRIBUTE},
```

Comment:
Why these values have been removed? I think they are needed. We need them for creating NRQL queries.

Reply:
They have not been removed but moved to `metricSetAttributes`